tlsfuzzer/runner.py

```diff
@@ -17,20 +17,21 @@ class ConnectionState(object):
     """
     Keeps the TLS connection state for sending of messages
 
-    @type msg_sock: L{tlslite.messagesocket.MessageSocket}
-    @ivar msg_sock: message level abstraction for TLS Record Socket
+    :ivar ~tlslite.messagesocket.MessageSocket msg_sock: message level
+        abstraction for TLS Record Socket
 
-    @ivar handshake_hashes: all handhsake messages hashed
+    :ivar handshake_hashes: all handhsake messages hashed
```

Comment:
no, canonically there is just on handshake messages hash, but it includes all messages exchanged to a given point, so "hashed" is correct

though there is a typo "handhsake" instead of "handshake" :)